EvseevSD/lab_2.cpp

```diff
@@ -0,0 +1,51 @@
+#include<iostream>
+#include<cstdlib>
+#include<conio.h>
+
+using namespace std;
+
+int main() {
+	const int masq1 = 8;
+	long int mas1[masq1], mas2[masq1], change , sum = 0;
+	int size2 = 0;
+	cout << "array with random elements :" << endl;
+	for (int i = 0; i < masq1; i++) {
+		mas1[i] = 0 + rand() % 2147483647;

```

Comment:
Expression "rand() % 2147483647" doesn't make any sense as rand() produces values in range 0 .. RAND_MAX where RANDMAX typicaly equals 0x7fff